apps/src/lab2/views/components/versionHistory/VersionHistoryDropdown.tsx

```diff
@@ -328,7 +328,6 @@ const VersionHistoryDropdown: React.FunctionComponent<
             ariaLabel: commonI18n.current(),
           },
         ]}
-        className={moduleStyles.latestTag}
```

Comment:
This style did not exist anymore, which is why there is no corresponding deletion in the scss file.